### ISSUE TITLE
it's a virus

### ISSUE THREAD

**Issue by 9tro** [kerememrerer/HWID-SPOOFER-UNDETECTED#1]
it's a virus big dawg. vs projects don't have that icon and don't have a ..sln lmao

**Comment by CallMeBurb2** [comment]
yep, tried on a vm just to see what happens and the sln file opens as a screensaver lmao
